eng/Packages.Data.props

```diff
@@ -207,7 +207,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(MSBuildProjectName.Contains('Azure.AI.Agents.Persistent'))">
-    <PackageReference Update="Microsoft.Extensions.AI.Abstractions" Version="9.6.0"/>
+    <PackageReference Update="Microsoft.Extensions.AI.Abstractions" Version="9.7.0"/>
```

Comment:
@stephentoub:  Version bump looks good.   Once the AI team has signed off on the client change, please mention me and I'll get this merged in.